src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,11 @@
+@function catalog-size (
```

Comment:
Maybe, let's use better naming (for example, catalog max width or something like that)